tests/scrolling_panning_zooming/test_zoom_menu_correlation.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from selenium.webdriver import Firefox
+from modules.browser_object import PanelUi
+from modules.page_object_generics import GenericPage
+from time import sleep
+
+
+@pytest.fixture()
+def test_case():
+    return "65064"
+
+
+TEST_PAGE = "https://en.wikipedia.org"
+
+
+def test_zoom_menu_correlation(driver: Firefox):
+    # Open a webpage to test zoom level
+
+    page = GenericPage(driver, url=TEST_PAGE)
+    sleep(1)
+
+    # Open the Firefox Menu panel and prepare to interact with zoom controls
+    panel = PanelUi(driver).open()
+    panel.open_panel_menu()
+
+    # Switch to Firefox's Chrome context for UI interaction
+    with driver.context(driver.CONTEXT_CHROME):
+        # Step 1: Zoom out twice to approximately 90%
+        panel.get_element("zoom-reduce").click()
+        sleep(2)
+
+        # Verify that the CSS zoom level reflects the 90% zoom out
+        css_zoom = page.get_css_zoom()
+        assert css_zoom == 0.9, f"Expected CSS zoom == 0.9, but got {css_zoom}"
```

Comment:
You can make this super stable if you implement the `wait_for_zoom` (or whatever you want to name it) mentioned in the page_base commentary

Reply:
Could not get that to work.